src/main/java/org/team199/robot2021/subsystems/Intake.java

```diff
@@ -77,5 +81,17 @@ public void doTheFlop() {
     public boolean isDeployed() {
         return deployed;
     }
+    public boolean isLimelightSearching() {
+        return isLimelightSearching;
+    }
+    public void changeServoStatus(boolean status) {
```

Comment:
Rename to `setLimelightSearching()`, since it isn't clear what a "servo status" is or what true or false would mean.